concepts/eventing-code-guidelines.md

```diff
@@ -389,6 +389,64 @@ MarkReady(apiRuleNew) // instead consider using another function that explicitly
 ```
 </details>
 
+### Logging Rules
```

Comment:
nit: Should we name it Logging Guidelines instead ? This would make it consistent with the rest of the document which talks about guidelines and not rules.
What do you think ?